### DISCUSSION TITLE
I want to allow players to be able to use containers while in combat

### DISCUSSION THREAD

**Discussion by ZepsiZola** [GriefPrevention/GriefPrevention#2004] [Ideas]
![image](https://user-images.githubusercontent.com/49735083/217856484-01ea88e1-66f9-4bb8-92aa-78c297a54f17.png)
![image](https://user-images.githubusercontent.com/49735083/217856640-2ede0347-6376-4329-9091-3b83cc1fb6c2.png)

I want to stop this from happening

**Comment by ZepsiZola** [comment]
So I suggest that the dev put a config in to allow server owners to allow players to use containers while in combat.

**Comment by bigpresh** [comment]
Setting the `PvP.CombatTimeoutSeconds` setting to 0 should prevent that happening - at the expense of also disabling punish on PvP logout and prevention of dropping items.

https://docs.griefprevention.com/configuration/#pvp-configuration

**Reply by ZepsiZola** [reply]
Thanks for this, but I was hoping I could find a way to simply just allow players to access inventory during pvp without also turning off features that I do like
